### DISCUSSION TITLE
OPDS: allow the use of file URIs

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1790] [Ideas]
In situations where the internet is not available and shared ebooks (especially textbooks) are stored in a network disk drive (i.e. a NAS + Wifi), it is useful to enable the use of OPDS feeds referenced via a file URI, and in which every reference is also a file URI.  

On Windows, file paths are formatted as `file:///c:/path/to/the%20file.txt`. 
On a Mac or Linux box, they are formatted as `file://hostname/path/to/the%20file.txt, or `file:///path/to/the%20file.txt` for localhost.

**Comment by danielweck** [comment]
I really think we need to discuss the ramifications of enabling local filesystem access. Moving to a discussion.

**Reply by danielweck** [reply]
See PR: https://github.com/edrlab/thorium-reader/pull/1549
